kpi/mixins/object_permission.py

```diff
@@ -441,23 +409,21 @@ def get_all_implied_perms(cls, for_instance=None):
         }
 
     @transaction.atomic
-    @kc_transaction_atomic
```

Comment:
@Guitlle Sorry I think I misled you on this one. 
I forgot about `set_kc_anonymous_permissions_xform_flags` in this method. In that case, we need to keep `@kc_transaction_atomic`